src/loader/renderer.ts

```diff
@@ -19,7 +19,7 @@ export const createContext = <Config extends PluginConfig>(
   id: string,
 ): RendererContext<Config> => ({
   getConfig: async () =>
-    window.ipcRenderer.invoke('ytmd:get-config', id) as Promise<Config>,
+    (await window.ipcRenderer.invoke('ytmd:get-config', id)) as Promise<Config>,
```

Comment:
Invalid type: If you use the await keyword here, the return value of await-invoke is `T`, not `Promise<T>`.